pkg/build/builder/sti.go

```diff
@@ -262,27 +261,27 @@ func (s *S2IBuilder) Build() error {
 			dockercfg.PushAuthType,
 		)
 		if authPresent {
-			glog.Infof("Using provided push secret for pushing %s image", pushTag)
+			glog.V(2).Infof("Using provided push secret for pushing %s image", pushTag)
 		} else {
-			glog.Infof("No push secret provided")
+			glog.V(2).Infof("No push secret provided")
 		}
-		glog.Infof("Pushing %s image ...", pushTag)
+		glog.V(0).Infof("Pushing %s image ...", pushTag)
```

Comment:
Anything we would always show would be zero.  I suppose we could argue
you might only want to see errors in some cases, but we don't set that
up yet.